.goreleaser.yml

```diff
@@ -49,3 +49,30 @@ nfpm:
   formats:
     - deb
     - rpm
+
+brew:
+  name: exo
+
+  github:
+    owner: pierre-emmanuelJ
+    name: homebrew-tap
+
+  # Git author used to commit to the repository.
+  # Defaults are shown.
+  commit_author:
+    name: pierre-emmanuelJ
+    email: pierre-emmanuel.jacquier@epitech.eu
+
+  # Your app's homepage.
+  # Default is empty.
+  homepage: "https://github.com/exoscale/egoscale/tree/master/cmd/exo"
+
+  # Your app's description.
+  # Default is empty.
+  description: "Manage easily your Exoscale infrastructure from the exo command-line"
```

Comment:
which one did you mean?

- _from the command-line_
- _with the exo command-line_